docker/hadoop/slave/conf/core-site.xml

```diff
@@ -0,0 +1,149 @@
+<!--Wed Dec  4 13:10:01 2019-->
```

Comment:
I don't think this file should be in this repository, it is better to automatically download a fresh copy of the hadoop conf directory a mount it in the docker container in every single run and thus make it fully dynamic.

Reply:
sounds good, will do